frontend/src/app/chart/chart.component.spec.ts

```diff
@@ -40,12 +40,26 @@ const generateNewData = (length: number) => {
   return channelData;
 };
 
+const generateMultiChannelData = (length: number) => {
+  const channelData: RecordsResponse[] = [];
+  const channels = ['SYS', 'PPX_ASYS', 'PP1800_SOC'];
+  for (const channel of channels) {
+    const recordsData: [number, number][] = [];
+    for (let index = 0; index < length; index++) {
```

Comment:
Looks like for this part, you can directly call generateSIngleChannelData() instead to copy paste the logic at here again.